package.json

```diff
@@ -6,6 +6,7 @@
   "bugs": "https://github.com/adobe/aio-cli-plugin-app/issues",
   "dependencies": {
     "@adobe/aio-cli-lib-app-config": "^1.1.0",
+    "@adobe/aio-cli-lib-app-config-next": "github:adobe/aio-cli-lib-app-config#cf70b5b185394593892bd695d5f9c41906759d8c",
```

Comment:
Note this will be updated to `"@adobe/aio-cli-lib-app-config-next" : "npm:@adobe/aio-cli-lib-app-config@^3.0.0" once https://github.com/adobe/aio-cli-lib-app-config/pull/27 is merged